lib/exfmt/ast/to_algebra.ex

```diff
@@ -354,12 +354,40 @@ defmodule Exfmt.Ast.ToAlgebra do
   end
 
   #
-  # Zero arity qualified function calls
+  # Zero arity type calls
+  #   e.g. String.t
+  #
+  def to_algebra({{:., _, [aliases, :t]}, _, []}, ctx) do
+    zero_arity_call_to_algebra(".t", aliases, ctx)
+  end
```

Comment:
Good catch. I meant to call this out in my pull request notes. I wasn't sure if t was reserved for type declarations. I think I can easily address this.